pipeline-elastic/samples/03-kaniko-push-dockerhub/0-resources.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: git-kaniko-helloworld
+  namespace: kaniko-tutorial
+spec:
+  type: git
+  params:
+  - name: url
+    value: https://github.com/mgreau/kaniko-helloworld.git
+---
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: image-kaniko-helloworld
+  namespace: kaniko-tutorial
+spec:
+  type: image
+  params:
+    - name: url
+      value: mgreau/kaniko-hello
```

Comment:
This is going to need to be parameterized to point to the users docker hub username. Perhaps we can grab it when we generate the config.json file in the changes I suggested.